app/src/main/res/layout/fragment_notification_guide.xml

```diff
@@ -0,0 +1,92 @@
+            app:layout_constraintBottom_toTopOf="@id/view_notification_guide_footer"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent">
+
+        </androidx.viewpager2.widget.ViewPager2>
+
+        <View
+            android:id="@+id/view_notification_guide_footer"
+            android:layout_width="0dp"
+            android:layout_height="56dp"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintBottom_toBottomOf="parent"
+            android:background="?attr/colorSurface" />
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/button_notification_guide_previous"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toStartOf="@id/page_indicator_notification_guide"
+            app:layout_constraintTop_toTopOf="@id/view_notification_guide_footer"
+            app:layout_constraintBottom_toBottomOf="@id/view_notification_guide_footer"
+            android:layout_marginStart="10dp"
+            app:layout_constraintHorizontal_bias="0.0"
+            app:layout_constrainedWidth="true"
+            android:text="@string/all_previous"
+            android:visibility="@{model.showPrevious ? View.VISIBLE : View.GONE}"
+            style="@style/Widget.Vilkku.Button.PageNav" />
+
+        <fi.thl.koronahaavi.widget.StepIndicatorView
+            android:id="@+id/page_indicator_notification_guide"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="@id/view_notification_guide_footer"
+            app:layout_constraintBottom_toBottomOf="@id/view_notification_guide_footer"
+            app:numSteps="5" />
+
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/button_notification_guide_next"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@id/page_indicator_notification_guide"
+            app:layout_constraintTop_toTopOf="@id/view_notification_guide_footer"
+            app:layout_constraintBottom_toBottomOf="@id/view_notification_guide_footer"
+            android:layout_marginEnd="10dp"
+            app:layout_constraintHorizontal_bias="1.0"
+            app:layout_constrainedWidth="true"
+            android:text="@string/all_next"
+            android:visibility="@{model.showNext ? View.VISIBLE : View.GONE}"
```

Comment:
On the final page a "close" button would be nice - moving a finger to the top left corner takes too much effort :)